swan-lake/development-tutorials/deployment-guide/k8s-deployment.md

```diff
@@ -0,0 +1,354 @@
+---
```

Comment:
@sm1990 Sometimes I am getting the following error when trying to render this page locally. Any idea?

<img width="1034" alt="image" src="https://github.com/user-attachments/assets/30d818ed-a18b-4323-b6a0-ed8f8be9f1be" />